Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN apt-get update && apt-get install -y \
 
 # Python Mergin client
 RUN python3 -m pip install --upgrade pip
-RUN pip3 install mergin-client==0.7.3 dynaconf==3.1
+RUN pip3 install mergin-client==0.7.3
```

Comment:
shall we start writing requirements file? it will be easier also to set up development

Reply:
I think we should go even further and unify approaches used in media-/dbsync and probably python client. E.g. use pipenv everywhere.

Reply:
sounds good to me